AirCasting/SessionViews/EditViewModal.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+struct EditViewModal: View {
+    
+    @Binding var showModalEdit: Bool
+    @State private var sessionName = ""
+    @State private var sessionTags = ""
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Spacer()
+            titleLabel
+                .padding(.bottom, 20)
+            createTextfield(placeholder: "Session name", binding: $sessionName)
```

Comment:
Dunno how it looks in Android so it's just a suggestion, feel free to ignore it. 
I think that it would be better to see the name&tags we're editing 🤔

Reply:
Agreed, for sure will focus on it (dunno if names but tags are in the designs) but will allow myself to do this as part of the logic - because it mostly depends on 😅